B827EBFFFE739593.json

```diff
@@ -3,11 +3,17 @@
     "gateway_ID": "B827EBFFFE739593",
     "servers": [
       {
-        "server_address": "router.eu.thethings.network",
+        "server_address": "router.asia-se.thethings.network",
         "serv_port_up": 1700,
         "serv_port_down": 1700,
         "serv_enabled": true
       }
+        {
```

Comment:
You're missing a comma here.

Reply:
Thank you so much for your kind reminder.